YDS-Storybook/SwiftUI/Foundation/TypoPageView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  TypoPageView.swift
+//  YDS-Storybook
+//
+//  Created by 박지윤 on 2023/08/22.
+//
+// swiftlint:disable nesting
+// swiftlint:disable line_length
+
+import SwiftUI
+import YDS_SwiftUI
+
+struct TypoListItem: View {
+    private enum Dimension {
+        enum Spacing {
+            static let vstack: CGFloat = 16
+            static let horizontal: CGFloat = 20
+            static let listItem: CGFloat = 2
+        }
+
+        enum Description {
+            enum Padding {
+                static let leading: CGFloat = -150
+                static let trailing: CGFloat = -55
+            }
+        }
+    }
+    let fontInfo: WrappedSwiftUITypo
+    var body: some View {
+        HStack {
+            VStack(spacing: 16) {
+                VStack(alignment: .leading, spacing: Dimension.Spacing.vstack) {
+                    ZStack {
+                        RoundedRectangle(cornerRadius: 2)
+                            .fill(YDSColor.monoItemBG)
+                            .frame(height: 24)
+                        HStack {
+                            Text(self.fontInfo.id)
+                                .font(YDSFont.caption1)
+                                .foregroundColor(YDSColor.monoItemText)
+                                .padding(.leading, 10)
+                            Spacer()
+                        }
+                    }
```

Comment:
```suggestion
                    HStack {
                        Text(self.fontInfo.id)
                            .font(YDSFont.caption1)
                            .foregroundColor(YDSColor.monoItemText)
                            .padding(.leading, 10)
                        Spacer()
                    }
                    .background {
                        RoundedRectangle(cornerRadius: 2)
                            .fill(YDSColor.monoItemBG)
                            .frame(height: 24)
                    }
```
ZStack을 쌓는 것 보다 background로 추가하는게 더 좋아보여요!